src/main/java/com/conveyal/gtfs/loader/Table.java

```diff
@@ -363,13 +364,18 @@ public Table (String name, Class<? extends Entity> entityClass, Requirement requ
         new StringField("geometry_type", REQUIRED)
     ).addPrimaryKey();
 
-    // https://github.com/MobilityData/gtfs-flex/blob/master/spec/reference.md#location_groupstxt-file-added
-    public static final Table LOCATION_GROUPS = new Table("location_groups", LocationGroup.class, OPTIONAL,
-        new StringField("location_group_id", REQUIRED),
-        new StringField("location_id", OPTIONAL).isReferenceTo(STOPS).isReferenceTo(LOCATIONS),
-        new StringField("location_group_name", OPTIONAL)
+    // https://github.com/MobilityData/gtfs-flex/blob/master/spec/reference.md#stop_areastxt-file-modified
+    public static final Table STOP_AREAS = new Table("stop_areas", StopArea.class, OPTIONAL,
+        new StringField("area_id", REQUIRED),
+        new StringField("stop_id", REQUIRED).isReferenceTo(STOPS).isReferenceTo(LOCATIONS)
     ).keyFieldIsNotUnique();
 
+    // https://github.com/MobilityData/gtfs-flex/blob/master/spec/reference.md#areastxt-no-change
+    public static final Table AREA = new Table("areas", Area.class, OPTIONAL,
+        new StringField("area_id", REQUIRED),
```

Comment:
Should this be `.isReferenceTo(STOP_AREA)`

Reply:
Yes, good spot. I have updated.